pep-0245.txt

```diff
@@ -3,7 +3,6 @@ Title: Python Interface Syntax
 Version: $Revision$
 Last-Modified: $Date$
 Author: Michel Pelletier <michel@users.sourceforge.net>
-Discussions-To: http://www.zope.org/Wikis/Interfaces
```

Comment:
I'd poked around a bit and could find one that actually resolved to the apparently intended page, and if did, it would mean I'd have to add a special case for this, since every other `Discusses-To` link/email both validates via the regex and parses and renders properly under the current logic in #2467 .

Reply:
The latest version of this on the Wayback Machine is [this from April 2006](https://web.archive.org/web/20060410173246/http://zope.org/Wikis/Interfaces/FrontPage) (that is after a redirect from [here](https://web.archive.org/web/20050415141542/http://www.zope.org/Wikis/Interfaces)).

Reply:
In general though, IMO it's worth keeping the original URI even if it's no longer available.

Reply:
Great, thanks for the find. Since this is a wiki page with resources rather than a discussion venue, would require special casing both the header parsing and validation logic, and we can provide some additional context thanks to your efforts, I moved it to a brief historical note at the top of the PEP, that includes the link to the archive as well as the original, and also links the archive of the now also-gone `interface-dev` mailing list linked there that contains much of the discussion leading to the PEP.